server/src/dtos/album.dto.ts

```diff
@@ -38,6 +38,7 @@ class AlbumUserCreateDto {
   userId!: string;
 
   @IsEnum(AlbumUserRole)
+  @ApiProperty({ enum: AlbumUserRole, enumName: 'AlbumUserRole' })
```

Comment:
It isn't necessary but you still prefer having it there?

Reply:
It is necessary actually. Otherwise it defaults to strings instead of a named enum.